pkg/pool-manager-utils/common.go

```diff
@@ -0,0 +1,271 @@
+/*
```

Comment:
we can name it to `pkg/controller-utils` as its a generics things can be used by cvr and cv controllers ? or have it inside controllers dir i.e. `pkg/controllers/utils`